src/stylesheets/core/_variables.scss

```diff
@@ -16,6 +16,7 @@ $lead-line-height:    1.7;
 
 $font-sans:           'Source Sans Pro', $helvetica;
 $font-serif:          'Merriweather', $georgia;
+$font-path:           '../fonts'
```

Comment:
So this should actually go in `core/_defaults.scss` and should be `$font-path: '../fonts' !default;`